client/app/core/config.js

```diff
@@ -1,20 +1,33 @@
 /* eslint angular/window-service: "off" */
+import { RootReducer } from '../reducers'
+import {createTransform, persistStore, autoRehydrate} from 'redux-persist'
```

Comment:
nitpick: inconsistent about spacing, did you consider using something like prettifier to unify the style?